src/app/components/cards/PostsList.jsx

```diff
@@ -267,6 +268,27 @@ class PostsList extends React.Component {
         const renderSummary = items =>
             items.map((item, i) => {
                 const every = this.props.adSlots.in_feed_1.every;
+                if (this.props.videoAdsEnabled && i === 4) {
+                    return (
```

Comment:
We can wrap this in `<center>` if we want it to be centered ...

Reply:
@roadscape think it would look better centered? I am undecided on that - it depends on a lot on the viewport size that the user has I suppose